hazelcast/src/main/java/com/hazelcast/client/impl/proxy/ClientScheduledExecutorProxy.java

```diff
@@ -433,4 +452,11 @@ private String extractNameOrGenerateOne(Object command) {
             throw rethrow(e);
         }
     }
+
+    private boolean isAutoDisposable(Object command) {
+        if (command instanceof AbstractTaskDecorator) {
```

Comment:
Same here.

Reply:
I think in that case this is handled at line 460